Арифметика/Арифметика/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Арифметика
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int x, y;
+            Console.WriteLine("Введите трёхзначное число");
+            //x = (Console.ReadLine());
+            x = int.Parse(Console.ReadLine());
+            y = ((x/10) + (x%10)*100);

```

Comment:
Все верно. Единственное замечание по стилю кода: отделяйте знаки операций пробелами. Так код легче читается. И внешние скобки можно не писать, они только загромождают оператор.
```suggestion
            y = (x / 10) + (x % 10) * 100;
```